src/lf_apply.rs

```diff
@@ -43,8 +44,9 @@ unsafe fn backup_lpf<BD: BitDepth>(
     };
     // The first stripe of the frame is shorter by 8 luma pixel rows.
     let mut stripe_h = ((64 as c_int) << (cdef_backup & sb128)) - 8 * (row == 0) as c_int >> ss_ver;
-    src = src.offset((stripe_h - 2) as isize * BD::pxstride(src_stride as usize) as isize);
-    if c.n_tc == 1 as c_uint {
+    src_offset = src_offset
+        .wrapping_add_signed((stripe_h - 2) as isize * BD::pxstride(src_stride as usize) as isize);
```

Comment:
We're not going anywhere near `usize::MAX`. To do `a += b` where `a` has type `usize` and `b` has type `isize`, is there a simpler/cleaner way than `a = (a as isize + b) as usize`?

Reply:
Hmm, they're both kinda ugly.  `a = (a as isize + b) as usize` is done because `b` can be negative, right?  I wish there was just a `.add_signed`.  I think, as unwieldy as `a = (a as isize + b) as usize` is, it might be a little bit better?  Since the `.wrapping` isn't there to confuse a reader as to why it's being done this way.  I'm not totally sure, though.

Reply:
I'm reverting to not using `.wrapping` to avoid unnecessary head scratching.